pkg/openwsn/contrib/openwsn.c

```diff
@@ -73,12 +74,10 @@ void* _radio_init_dev(void)
     void* dev = NULL;
     /* avoid cppcheck style (redundantAssignment)*/
     (void) dev;
-#ifdef MODULE_OPENWSN_RADIO_NETDEV
     #ifdef MODULE_AT86RF2XX
-        dev = &at86rf2xx_dev.netdev.netdev;
-        at86rf2xx_setup(&at86rf2xx_dev, &at86rf2xx_params[0], 0);
+        dev = &at86rf2xx_dev;
+        at86rf2xx_init_event(&at86rf2xx_bhp, &at86rf2xx_params[0], &at86rf2xx_dev, NULL, EVENT_PRIO_HIGHEST);
```

Comment:
Too many arguments

Reply:
I changed the signature of the `init` function and forgot to update those. I just amended directly. It should work now.